common/mlsdbserialisation.cpp

```diff
@@ -11,43 +11,63 @@
 */
 
 #include "mlsdbserialisation.h"
+#include "mccmapping.h"
 
-QDataStream &operator<<(QDataStream &out, const MlsdbCoords &coords)
-{
-    out << coords.lat << coords.lon;
-    return out;
+QString stringForMlsdbCellType(MlsdbCellType type) {
+    switch (type) {
+        case MLSDB_CELL_TYPE_LTE: return QLatin1String("LTE");
+        case MLSDB_CELL_TYPE_GSM: return QLatin1String("GSM");
+        case MLSDB_CELL_TYPE_UMTS: return QLatin1String("UMTS");
+        default: return QLatin1String("OTHER");
+    }
 }
 
-QDataStream &operator>>(QDataStream &in, MlsdbCoords &coords)
-{
-    in >> coords.lat >> coords.lon;
-    return in;
+quint64 getMlsdbUniqueCellId(MlsdbCellType cellType, quint32 cellId, quint32 locationAreaCode, quint16 mcc, quint16 mnc) {
```

Comment:
Nitpick: by qt coding style, the method starting braces to their own lines. Applies to other cases in the PR too.